src/MarkdownTextInput.tsx

```diff
@@ -40,14 +41,14 @@ function initializeLiveMarkdownIfNeeded() {
   if (!global.jsi_setMarkdownRuntime) {
     throw new Error('[react-native-live-markdown] global.jsi_setMarkdownRuntime is not available');
   }
-  workletRuntime = createWorkletRuntime('LiveMarkdownRuntime');
+  workletRuntime = createWorkletRuntime({name: 'LiveMarkdownRuntime'});
   global.jsi_setMarkdownRuntime(workletRuntime);
   initialized = true;
 }
 
 function registerParser(parser: (input: string) => MarkdownRange[]): number {
   initializeLiveMarkdownIfNeeded();
-  const shareableWorklet = makeShareableCloneRecursive(parser) as unknown as ShareableRef<WorkletFunction<[string], MarkdownRange[]>>;
+  const shareableWorklet = createSerializable(parser) as unknown as SerializableRef<WorkletFunction<[string], MarkdownRange[]>>;
```

Comment:
Is it possible to remove the part with `as unknown` or even the whole cast now?

Also, let's rename `shareableWorklet` to `serializableWorklet` in this PR or a separate PR.